studio/Dockerfile

```diff
@@ -7,19 +7,19 @@
 #    docker builder prune
 
 
-FROM node:14 as base
+FROM node:14-alpine as base
 WORKDIR /usr/src/app
 # Do `npm ci` separately so we can cache `node_modules`
 # https://nodejs.org/en/docs/guides/nodejs-docker-webapp/
 COPY package*.json ./
-RUN npm clean-install
+RUN npm clean-install && npm cache clean --force
```

Comment:
Its probably cleaner to move this line inside the dev and production stages. For dev we can do npm ci and for prod npm install --only=prod. That way we dont need to prune, clean up webpack caches etc

Reply:
We still need to install development dependencies to run `next build`. So we can either stick with the current solution where we install all dependencies and prune the development ones later, or we can run `next build` in a separate multi-stage build step, and then copy the built `.next` files into another stage where we only install production dependencies.

Reply:
could we do something like this:
```
FROM node:14-slim as base
RUN npm ci && npm run build

FROM node:14-slim as production
COPY --from=base /usr/src/app/.next /usr/src/app/.next
WORKDIR /usr/src/app
RUN npm i -g next && \
    rm -rf ./next/cache && \
    # or the alpine equivalent
    apt-get remove npm && \
    next start
```
From my understanding of the bundling from the [deployment docs](https://nextjs.org/docs/deployment), server-side deps are already bundled so the only dep required will be the next cli.

Do correct me if i'm wrong.

Reply:
Interesting, I'll test it out sometime later.